test/e2e/apimachinery/watch.go

```diff
@@ -325,7 +325,7 @@ var _ = SIGDescribe("Watchers", func() {
 	   for events received from the first watch, initiated at the resource version of the event, and checking that all
 	   resource versions of all events match. Events are produced from writes on a background goroutine.
 	*/
-	It("should receive events on concurrent watches in same order", func() {
+	framework.ConformanceIt("should receive events on concurrent watches in same order", func() {
```

Comment:
Metadata needs Release (1.15) too: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-comment-metadata

Reply:
And one last thing - you need to add the test to conformance.txt

Reply:
Thanks, fixed.